src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
-public class Application {
+import java.util.Arrays;
+
+public class Application extends Lottery {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < 3; i++) {
+            balls[i] = new Lottery().getRandomBall();
```

Comment:
imagine such situation in real life - when you want to get three balls from lottery, you dont need to build 3 separate lotteries. 
create lottery once and reuse it